src/Event/Events.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Event;
+
+final class Events
+{
+    const PRE_READ = 'api_platform.pre_read';
+    const POST_READ = 'api_platform.post_read';
+
+    const PRE_WRITE = 'api_platform.pre_write';
+    const POST_WRITE = 'api_platform.post_write';
+
+    const PRE_SERIALIZE = 'api_platform.pre_serialize';
+    const POST_SERIALIZE = 'api_platform.post_serialize';
+
+    const PRE_DESERIALIZE = 'api_platform.pre_deserialize';
+    const POST_DESERIALIZE = 'api_platform.post_deserialize';
+
+    const PRE_RESPOND = 'api_platform.pre_respond';
```

Comment:
These events should be removed as they're not related to the resource.

Reply:
Sorry, should I remove that event [or move it](https://github.com/api-platform/core/pull/2329#discussion_r239877920)?

Reply:
Only keep the events mentioned in https://github.com/api-platform/core/pull/2329#discussion_r239877920 (in the correct order).

Reply:
I see you've already removed the "Respond" events, so it's already correct.